frontend/catalyst/third_party/oqc/src/oqc_python_module.cpp

```diff
@@ -22,30 +22,31 @@
 #include "Exception.hpp"
 
 std::string program = R"(
-import os
-from qcaas_client.client import OQCClient, QPUTask, CompilerConfig
-from qcaas_client.config import QuantumResultsFormat, Tket, TketOptimizations
-optimisations = Tket()
-optimisations.tket_optimizations = TketOptimizations.DefaultMappingPass
-
-RES_FORMAT = QuantumResultsFormat().binary_count()
 
 try:
+    import os
     email = os.environ.get("OQC_EMAIL")
     password = os.environ.get("OQC_PASSWORD")
     url = os.environ.get("OQC_URL")
+    if not all([email, password, url]):
+        raise ValueError("OQC credentials not found in environment variables")
```

Comment:
For error messages, the rule of thumb is suggest user actions whenever possible! 
```suggestion
        raise ValueError("OQC credentials not found in environment variables. Please set the environment variables `OQC_EMAIL`, `OQC_PASSWORD` and `OQC_URL`.")
```